apps/frontend/src/pages/my-wheel/my-wheel.tsx

```diff
@@ -0,0 +1,14 @@
+import { Sidebar } from "~/libs/components/components.js";
+
+import styles from "./styles.module.css";
+
+const MyWheel: React.FC = () => {
+	return (
+		<div className={styles["pageContainer"]}>
+			<Sidebar />
+			<div />
```

Comment:
Do we need this empty div?

Reply:
Not really, it is just for demonstration purposes. The rest of the page should be in place of this div

Reply:
You can remove for now